app/controllers/qa_controller.rb

```diff
@@ -96,25 +97,26 @@ def nested_options(filter_group)
   end
   helper_method :nested_options
 
-  def last_page?
-    page == facets.length
-  end
-  helper_method :last_page?
-
   def next_page
     page + 1
   end
   helper_method :next_page
 
-  def next_page_url
-    return qa_config["finder_base_path"] if last_page?
+  def finder_page?
+    page == facets.length + 1
+  end
 
+  def next_page_url
     qa_config["base_path"]
   end
   helper_method :next_page_url
 
+  def redirect_to_finder
+    redirect_to qa_config["finder_base_path"] + '?' + filtered_params.to_query and return
```

Comment:
Is the `and return` needed?

Reply:
I thought that this would ensure that we stopped controller execution, but as there is no other option it doesn't seem to make any difference in this case, and also resolves the Rubocop problem @AlanGabbianelli wrote about below.

I'll remove it 👍